src/xdp/program.h

```diff
@@ -75,6 +76,18 @@ XdpProgramCanXskBypass(
     _In_ XDP_RX_QUEUE *RxQueue
     );
 
+NTSTATUS
+XdpProgramCreate(
+    _Out_ XDP_PROGRAM_OBJECT **NewProgramObject,
+    _In_ const XDP_PROGRAM_OPEN *Params,
```

Comment:
```
##[error]src\xdp\program.h(82,16): Error C2220: the following warning is treated as an error
    41>D:\a\1\s\src\xdp\program.h(82,16): error C2220: the following warning is treated as an error [D:\a\1\s\test\pktfuzz\pktfuzz.vcxproj]
```
Looks like the user mode fuzzer doesn't like this.

Reply:
Link: https://mscodehub.visualstudio.com/WindowsXDP/_build/results?buildId=423050&view=logs&j=dd755d80-e2a8-5bc6-1364-e03f69bfc23b&t=e52c1f6d-48cd-5bea-e390-3ebdba0d6742